src/test/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/LicenceFunctionsKtTest.kt

```diff
@@ -83,6 +89,78 @@ class LicenceFunctionsKtTest {
         sedChanged = true,
         tussdChanged = false,
         tusedChanged = false,
+        pedChanged = false,
+        npdChanged = false,
+        prrdChanged = false,
+        isMaterial = true,
+      ),
+    )
+  }
+
+  @Test
+  fun `Sentence Changes should return material change for SED when paroleEligibilityDate is updated`() {
```

Comment:
I think there's a bit of a misnomer here, as the name of each of the new tests references `should return material change for SED`, but `SED` isn't changing in any of these tests